docs/development/Defines.md

```diff
@@ -0,0 +1,531 @@
+# Defines
+
+Betaflight uses `#DEFINE` syntax to include software components to support additional features at build time.
+
+A basic set of defines is included in the config file for the board. At build time, the user can add extra defines in the the Configurator's Firmware Flasher Tab.
+
+When selecting GPS, Baro, etc from the `Other Options` list the required additional defines are added automatically. For example, if `BARO` is requested by the user, the `USE_BARO` define, and all supported baro hardware defines are automatically included.
+
+Additional defines may be added:
+
+- to a cloud build, by entering the name of the define, \_without the USE\_\_ prefix, separated by spaces, in the `Custom Defines` list, e.g. `ANA EXTI`.
```

Comment:
fix two of the underscored to italics, not rendering.
```suggestion
- to a cloud build, by entering the name of the define, _without the USE\__ prefix, separated by spaces, in the `Custom Defines` list, e.g. `ANA EXTI`.
```

Reply:
Thanks I'll just not italicise that area for simplicity